docs/architecture/seo.md

```diff
@@ -1,46 +1,47 @@
 
-URLs for non-content pages are configurable in Spartacus. This is mainly related to product and category pages. Attributes, such the product name can be configured to be part of the URL. 
+## Configurable URLs
 
-i.e. the default configuration for a product page is storefront.com/product/1234, but customers can improve the URL in order to add the product or cateogry title to the URL.
+You can configure URLs for content pages by using a CMS page label in the CMS (back end). These page labels cannot be localized.
 
-Configurable URLs will help to improve SEO in general, but also help to migrate an existing solution to Spartacus; customers can keep their existing URLs and configure the equivalent URLs in spartacus. 
+You can configure URLs for non-content pages in Spartacus. These are mainly related to product and category pages. You can configure attributes, such as the product name, to be part of the URL. For example, the default configuration for a product page is `storefront.com/product/1234`, but you can modify the URL to include the product or category title. 
 
-**notes**: 
-- the product code is used for resolving the product data from the backend. The rest of the URL can be configured for SEO reasons. 
-- some customers have product titles with special characters or even user characters that won't wrok (i.e. using a slash in the code or title). This migh require special treatment of the attributes before/after they're used in the URL. 
+Configurable URLs help to improve SEO in general, but can also be used to help migrate an existing solution to Spartacus: customers can keep their existing URLs, and configure the equivalent URLs in Spartacus.
 
+**Note**: The product code is used for resolving the product data from the back end. The rest of the URL can be configured for SEO purposes.
+
+**Note**: Some customers have product titles with special characters that will not work (for example, having a slash in the code or title). This might require special treatment of the attributes before or after they are used in the URL.
 
 ## Indexable Pages
-Server Side Rendering (SSR) is a technique to render the javascript logic server side, and provide rich content in the response. The SSR response contains the full HTML that is required by crawlers (Google, Facebook, etc.) in order to index or retrieve data from the response. 
 
-SSR is provided by Spartacus and is planned to be a default deployment option at Commerce Cloud.
+Server-side rendering (SSR) is a technique that renders the JavaScript logic on the server side, and then provides rich content in the response. The SSR response contains the full HTML that is required by web crawlers to index or retrieve data from the response.
+
+SSR is provided by Spartacus, and is planned to be a default deployment option in SaaS Commerce Cloud.
```

Comment:
if we have SSR specific docs, we should have a link.

Reply:
I added links to the SSR coding guidelines (which themselves contain a link to "Configurable State Management" which discusses SSR), as well as to "Controlling Server Side Rendering".